src/data/uma-nickname.csv

```diff
@@ -88,17 +88,17 @@
 "F・チャンピオン","F・Champion"
 "トリプルティアラ","Triple Tiara"
 "直線番長","Straight Liner"
-"名バを訪ねて",""
+"名バを訪ねて","Look Back on Great Horses"
 "ジンクスブレイカー","Jinx Breaker"
 "スピードクイーン","Speed Queen"
 "年末の覇者","Year-End Champion"
 "浪花節","Naniwabushi"
-"変則二冠ウマ娘",""
+"変則二冠ウマ娘","Double Irregular Crown"
```

Comment:
Why the "Irregular" isn't at the front? Irregular Double Crown seems fit better than Double Irregular Crown :/

Reply:
Yes, I believe `Irregular Double Crown` is the better here, since [Double Crown](https://ja.wikipedia.org/wiki/二冠馬) is a thing, and this is the irregular version of that.

I'm not sure if `Irregular` is the best translation for `変則` though.

Reference: https://ja.wikipedia.org/wiki/二冠馬#変則二冠馬

Reply:
Well , Rough translation for 【変則】are "Irregularity" and "Anomaly" so there are kind of relation to something that "weird" also i'm not pretty sure either.

Reply:
`Unorthodox` is another word I could think of, but I guess `Irregular Double Crown` is good too

Reply:
```suggestion
"変則二冠ウマ娘","Irregular Double Crown"
```

Reply:
Agreed.